lib/validator.js

```diff
@@ -192,13 +192,13 @@ Specberus.prototype.getSotDSection = function () {
     if (!$startH2 || !$endH2) return null;
     var started = false;
     $startH2.parent().children().each(function () {
-        if ($startH2[0] === this[0]) {
+        if ($startH2[0] === this) {
             started = true;
             return;
         }
         if (!started) return;
-        if ($endH2[0] === this[0]) return false;
-        $div.append(this.clone());
+        if ($endH2[0] === this) return false;
+        $div.append($(this).clone());
     });
```

Comment:
In jQuery, `each` sets `this` to the DOM element (not its jQuery wrapper). Clearly whacko wasn't doing that and I worked around it (I don't recall the issue, but it looks that way). It seems that whacko is now working a lot more like jQuery. This could potentially bite for any other instance of `each`.